env.example

```diff
@@ -1,6 +1,21 @@
-export LOCATION=eastus
+export LOCATION=westeurope
```

Comment:
Can we change this back to `eastus`? I believe `westeurope` was used because of some issues with the region during the work.

Reply:
There are still issues in `eastus` atm fyi